src/layouts/index.js

```diff
@@ -4,13 +4,26 @@ import Helmet from 'react-helmet'
 
 import { Navbar, Footer } from '../components'
 import {Grommet, grommet} from 'grommet';
+import { metro } from "grommet-controls";
+import { deepMerge } from "grommet/utils";
 
 import './reset.css'
 import './normalize.css'
 import './index.css'
 
+// const customTheme = deepMerge(metro, {
+//   global: {
+//     breakpoints: {
+//       small: {
+//         value: 300
+//       },
+//       medium: undefined,
+//     }
+//   }
+// });
```

Comment:
Don't like so much commented code. Why is this here?

Reply:
it's the only way to change the theme, just let it be for a future purpose

Reply:
Fair enough